modules/installation-aws-about-government-region.adoc

```diff
@@ -0,0 +1,24 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_aws/installing-aws-government-region.adoc
+
+[id="installation-aws-about-government-region_{context}"]
+= AWS Government regions
+
+{product-title} supports deploying a cluster to
+link:https://aws.amazon.com/govcloud-us[AWS GovCloud (US)] regions. AWS GovCloud
+is specifically designed for US government agencies at the federal, state, and
+local level, as well as contractors, educational institutions, and other US
+customers that must run sensitive workloads in the cloud.
+
+These regions do not have published {op-system-first} AMIs to select, so you
+must upload a custom AMI that belongs to that region.
+
+The following AWS GovCloud partitions are supported:
+
+* `us-gov-west-1`
+* `us-gov-east-1`
+
+The AWS GovCloud region and custom AMI must be manually configured in the
+`install-config.yaml` file since {op-system} AMIs are not provided by Red Hat
+for those regions.
```

Comment:
In this section, do we need to clarify that only private cluster is supported in US-Gov regions?

Reply:
@abhinavdahiya  could you PTAL and provide feedback for this? Thanks

Reply:
Workloads running in GovCloud regions can be internet facing, but the challenge is that public zones are not supported in Route53 in GovCloud, so the user would need to define their own public DNS entries in the case of GovCloud. Deploying a private cluster, should work fully though because private zones are supported in Route53 in GovCloud.

Reply:
get it, thanks @jaredhocutt

Reply:
I think @jaredhocutt is correct that the gov cloud region does not allow for creating public route53 zones.

So i think it would make sense we put this information here, that currently we can only support internal clusters as users cannot create these public route53 zones.